build-image-manual.sh

```diff
@@ -15,7 +18,7 @@ TAG=$2
 IMAGE=$IMAGE_USER/$IMAGE_PREFIX$DOCKERFILE
 
 log() {
-    echo -e "\e[35m[build-image.sh]\e[39m $@"
+    echo "\e[35m[build-image-manual.sh]\e[39m $@"
```

Comment:
Why remove `-e`?  It's  normally needed for escape sequences.

Reply:
The "-e" was being printed to my terminal and when I removed it, it looked fine